modules/udf/variables.tf

```diff
@@ -21,8 +21,3 @@ variable "dataset_id" {
 variable "project_id" {
   description = "Project ID that contains the dataset"
 }
-
-variable "add_udfs" {
```

Comment:
I'd rather keep this variable. Even though we're not bundling it ourselves any more, downstream customer modules might and this variable provides flexibility.